os/net/packetbuf.h

```diff
@@ -265,10 +265,10 @@ int               packetbuf_set_addr(uint8_t type, const linkaddr_t *addr);
 const linkaddr_t *packetbuf_addr(uint8_t type);
 
 /**
- * \brief      Checks whether the current packet is a broadcast.
- * \retval 0   iff current packet is not a broadcast
+ * \brief       Checks whether the current packet is a broadcast.
+ * \retval true iff the current packet is not a broadcast
```

Comment:
Now the comment says that the function returns <code>true</code> "iff the current packet is _not_ a broadcast".

Reply:
Fixed comment